src/modules/aws_iam/migration/1671659720979-aws_iam.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
Btw, things are failing right now because the old migration was accidentally not deleted.